app/models/backup_code_configuration.rb

```diff
@@ -7,6 +7,9 @@ class BackupCodeConfiguration < ApplicationRecord
 
   belongs_to :user
 
+  attr_accessor :skip_legacy_encryption
+  alias_method :skip_legacy_encryption?, :skip_legacy_encryption
+
   def self.unused
     where(used_at: nil)
```

Comment:
I know this isn't new code but it will need a [user_id, used_at] index

Reply:
Scratch this.  We are deleting the user's codes when we regen.  Nothing like forgetting what I wrote...